config.json

```diff
@@ -839,7 +852,12 @@
       "core": false,
       "difficulty": 8,
       "slug": "simple-cipher",
-      "topics": null,
+      "topics": [
+        "strings",
+        "cryptography",
+        "security",
+        "mathematics"
```

Comment:
I would add `randomness` and `exception_handling` as well.

Reply:
Yes, exception_handling can be used also but at the same time, we can check the input manually without worrying about the exception. Actually, I didn't think in a language-specific way.

Reply:
Again the [tests](https://github.com/exercism/java/blob/master/exercises/simple-cipher/src/test/java/SimpleCipherStepThreeTest.java) expect an exception to be thrown so you have to use exceptions. These are meant to be language specific topics for doing the exercises on the java track :)